### ISSUE TITLE
Add detailed example workflows for spatial optimization in the README

### ISSUE THREAD

**Issue by apawlik** [manaakiwhenua/LUMASS#5]

**Comment by heralex** [comment]
Yes, I certainly want to integrate the [Optimisation HowTo](https://manaakiwhenua.github.io/LUMASS/assets/material/OptimisationHowTo_1.2.zip) content to the [Documentation](https://manaakiwhenua.github.io/LUMASS/). What is still missing is a sample model of how to integrate the optimisation into a LUMASS modelling workflow and how to make the *.los files more dynamic. There will also be information on the new GA-based optimisation that will be available soon.
